failure-accrual/README.md

```diff
@@ -2,14 +2,46 @@
 
 This directory contains a docker-compose environment that runs a demo that you
 can use to test [linkerd's failure accrual settings](
-https://linkerd.io/features/circuit-breaking/#failure-accrual).
+https://linkerd.io/features/circuit-breaking/#failure-accrual). The results of
+the demo are discussed in much more detail in Buoyant's blog post, [Making
+microservices more resilient with advanced circuit breaking](
+http://blog.buoyant.io/2017/01/13/making-microservices-more-resilient-with-advanced-circuit-breaking/).
 
 ## Startup
 
-To start linkerd preconfigured with 3 routers, routing to 15 go backends, using
-different failure accrual settings for each router, generating traffic via
-[slow_cooker](https://github.com/BuoyantIO/slow_cooker), and exporting stats to
-prometheus and grafana, run:
+The [`docker-compose.yml`](docker-compose.yml) file that's included in this
+directory is configured to run the demo, which consists of all of the following
+applications:
+
+* **linkerd**: linkerd is configured via the provided [`linkerd.yml`](
+linkerd.yml), which specifies 3 separate routers, running on ports 4141, 4142,
+and 4143, each configured with different failure accrual policies, and each
+routing traffic to separate backend clusters. The linkerd config also specifies
+a 4th router running on port 4140, which evenly distributes its requests over
+the other three routers, which is useful for traffic generation.
+
+* **3 backend clusters**: Each cluster runs 5 instances of the go web server
+that's defined in [`server.go`](server.go). 4 of the 5 instances in each cluster
+return responses immediately with 100% success rate. 1 instance simulates a 60%
+success rate, and adds 100 milliseconds of latency to failing requests.
+
+* **slow_cooker**: Traffic to linkerd is generated using [slow\_cooker](
+https://github.com/BuoyantIO/slow_cooker). slow\_cooker is configured to send
+300 requests per second to the linkerd router that is running on port 4140,
+which evenly distributes the requests to the other 3 routers, such that each
+router receives roughly 100 requests per second.
+
+* **prometheus**: linkerd and the backend instances expose metrics data in a
+format that can be read by [Prometheus](https://prometheus.io/). Prometheus
+metrics collection is configured in [`prometheus.yml`](prometheus.yml), which
+scrapes all metrics from all processes every 5 seconds.
+
+* **grafana**: Collected metrics are displayed on a dashboard using [Grafana](
+http://grafana.org/). The grafana container is preconfigured with the failure
+accrual comparison dashboard that is defined in [`grafana.json`](grafana.json).
+
+Using the magic of docker-compose, you can start all of the containers outlined
+above with the following docker-compose commands:
```

Comment:
i like readme's that start off with the One Great Command To Boot Everything. consider moving it back up to the top.